### ISSUE TITLE
Can't install the package + Laravel 5.6.*

### ISSUE THREAD

**Issue by thanhnd2812** [antonioribeiro/google2fa-laravel#39]
I'm using Laravel 5.6.* version and installing the package from composer, but got the errors:

```
Problem 1
    - Conclusion: don't install pragmarx/google2fa-laravel v0.2.0
    - Conclusion: remove paragonie/random_compat v9.99.99
    - Installation request for pragmarx/google2fa-laravel ^0.2.0 -> satisfiable by pragmarx/google2fa-laravel[0.2.x-dev, v0.2.0].
    - Conclusion: don't install paragonie/random_compat v9.99.99
    - pragmarx/google2fa-laravel 0.2.x-dev requires pragmarx/google2fa ~3.0 -> satisfiable by pragmarx/google2fa[v3.0.0, v3.0.1, v3.0.2].
    - pragmarx/google2fa v3.0.0 requires paragonie/random_compat ~1.4|~2.0 -> satisfiable by paragonie/random_compat[v1.4.0, v1.4.1, v1.4.2, v1.4.3, v1.x-dev, v2.0.0, v2.0.1, v2.0.10, v2.0.11, v2.0.12, v2.0.13, v2.0.14, v2.0.15, v2.0.16, v2.0.17, v2.0.2, v2.0.3, v2.0.4, v2.0.5, v2.0.6, v2.0.7, v2.0.8, v2.0.9].
    - pragmarx/google2fa v3.0.1 requires paragonie/random_compat ~1.4|~2.0 -> satisfiable by paragonie/random_compat[v1.4.0, v1.4.1, v1.4.2, v1.4.3, v1.x-dev, v2.0.0, v2.0.1, v2.0.10, v2.0.11, v2.0.12, v2.0.13, v2.0.14, v2.0.15, v2.0.16, v2.0.17, v2.0.2, v2.0.3, v2.0.4, v2.0.5, v2.0.6, v2.0.7, v2.0.8, v2.0.9].
    - pragmarx/google2fa v3.0.2 requires paragonie/random_compat >=1 <9.99 -> satisfiable by paragonie/random_compat[1.0.10, 1.0.2, 1.0.3, 1.0.4, 1.0.5, 1.0.6, 1.0.7, 1.0.8, 1.0.9, 1.1.0, 1.1.1, 1.1.2, 1.1.3, 1.1.4, 1.1.5, 1.1.6, v1.0.0, v1.0.1, v1.2.0, v1.2.1, v1.2.2, v1.2.3, v1.3.0, v1.3.1, v1.4.0, v1.4.1, v1.4.2, v1.4.3, v1.x-dev, v2.0.0, v2.0.1, v2.0.10, v2.0.11, v2.0.12, v2.0.13, v2.0.14, v2.0.15, v2.0.16, v2.0.17, v2.0.2, v2.0.3, v2.0.4, v2.0.5, v2.0.6, v2.0.7, v2.0.8, v2.0.9].
    - Can only install one of: paragonie/random_compat[v1.4.0, v9.99.99].
    - Can only install one of: paragonie/random_compat[v1.4.1, v9.99.99].
    - Can only install one of: paragonie/random_compat[v1.4.2, v9.99.99].
    - Can only install one of: paragonie/random_compat[v1.4.3, v9.99.99].
    - Can only install one of: paragonie/random_compat[v1.x-dev, v9.99.99].
    - Can only install one of: paragonie/random_compat[v2.0.0, v9.99.99].
    - Can only install one of: paragonie/random_compat[v2.0.1, v9.99.99].
    - Can only install one of: paragonie/random_compat[v2.0.10, v9.99.99].
    - Can only install one of: paragonie/random_compat[v2.0.11, v9.99.99].
    - Can only install one of: paragonie/random_compat[v2.0.12, v9.99.99].
    - Can only install one of: paragonie/random_compat[v2.0.13, v9.99.99].
    - Can only install one of: paragonie/random_compat[v2.0.14, v9.99.99].
    - Can only install one of: paragonie/random_compat[v2.0.15, v9.99.99].
    - Can only install one of: paragonie/random_compat[v2.0.16, v9.99.99].
    - Can only install one of: paragonie/random_compat[v2.0.17, v9.99.99].
    - Can only install one of: paragonie/random_compat[v2.0.2, v9.99.99].
    - Can only install one of: paragonie/random_compat[v2.0.3, v9.99.99].
    - Can only install one of: paragonie/random_compat[v2.0.4, v9.99.99].
    - Can only install one of: paragonie/random_compat[v2.0.5, v9.99.99].
    - Can only install one of: paragonie/random_compat[v2.0.6, v9.99.99].
    - Can only install one of: paragonie/random_compat[v2.0.7, v9.99.99].
    - Can only install one of: paragonie/random_compat[v2.0.8, v9.99.99].
    - Can only install one of: paragonie/random_compat[v2.0.9, v9.99.99].
    - Can only install one of: paragonie/random_compat[1.0.10, v9.99.99].
    - Can only install one of: paragonie/random_compat[1.0.2, v9.99.99].
    - Can only install one of: paragonie/random_compat[1.0.3, v9.99.99].
    - Can only install one of: paragonie/random_compat[1.0.4, v9.99.99].
    - Can only install one of: paragonie/random_compat[1.0.5, v9.99.99].
    - Can only install one of: paragonie/random_compat[1.0.6, v9.99.99].
    - Can only install one of: paragonie/random_compat[1.0.7, v9.99.99].
    - Can only install one of: paragonie/random_compat[1.0.8, v9.99.99].
    - Can only install one of: paragonie/random_compat[1.0.9, v9.99.99].
    - Can only install one of: paragonie/random_compat[1.1.0, v9.99.99].
    - Can only install one of: paragonie/random_compat[1.1.1, v9.99.99].
    - Can only install one of: paragonie/random_compat[1.1.2, v9.99.99].
    - Can only install one of: paragonie/random_compat[1.1.3, v9.99.99].
    - Can only install one of: paragonie/random_compat[1.1.4, v9.99.99].
    - Can only install one of: paragonie/random_compat[1.1.5, v9.99.99].
    - Can only install one of: paragonie/random_compat[1.1.6, v9.99.99].
    - Can only install one of: paragonie/random_compat[v1.0.0, v9.99.99].
    - Can only install one of: paragonie/random_compat[v1.0.1, v9.99.99].
    - Can only install one of: paragonie/random_compat[v1.2.0, v9.99.99].
    - Can only install one of: paragonie/random_compat[v1.2.1, v9.99.99].
    - Can only install one of: paragonie/random_compat[v1.2.2, v9.99.99].
    - Can only install one of: paragonie/random_compat[v1.2.3, v9.99.99].
    - Can only install one of: paragonie/random_compat[v1.3.0, v9.99.99].
    - Can only install one of: paragonie/random_compat[v1.3.1, v9.99.99].
    - Installation request for paragonie/random_compat (locked at v9.99.99) -> satisfiable by paragonie/random_compat[v9.99.99].
```

Any suggestions?

Thanks in advance!!!

**Comment by sammax** [comment]
paragonie/random_compat is locked at 9.99.99 and for some inane reason `composer require` does not downgrade dependencies. `composer update` however does, so you can first `composer require --no-update pragmarx/google2fa-laravel` and then `composer update`. This will however update all your dependencies.

**Comment by Shandur** [comment]
as a solution you can just install this package manually as project's dependency: paragonie/random_compat:"^2.0"

**Comment by Shandur** [comment]
@thanhnd2812 I found another one solution, there is an explanation https://github.com/paragonie/random_compat#version-99999

tl;dr: you just need to add this to your composer.json

`
    "replace": {
        "paragonie/random_compat": "9.99.99"
    }
`

**Comment by thanhnd2812** [comment]
@Shandur Thanks! Solved!!!
